toolchain/syno-qoriq-6.2.4-rust/Makefile

```diff
@@ -2,7 +2,7 @@ PKG_ARCH = qoriq
 PKG_NAME = rust-$(PKG_ARCH)
 PKG_VERS = 1.82.0
 PKG_EXT = txz
-PKG_DIST_TARGET = $(PKG_VERS)-powerpc-unknown-linux-gnuspe
+PKG_DIST_TARGET = $(PKG_VERS)-powerpc-unknown-linux-gnuspe-spksrc
```

Comment:
I don't believe that this is the correct rename strategy:

1. The original filename didn't change so this will result in a file not found.
2. The issue is the downloaded file. Once downloaded it should be renamed to match the syntax of https://rust-lang.github.io/rustup/overrides.html#toolchain-file-settings. In theory, it needs a prefix before the version number (e.g. `stable|beta|nightly`)

Reply:
@th0ma7 what was the reason for this renaming?
- is it related to #6476?
- your rename can't work, because the PKG_DIST_TARGET is the name of the file to download
- but we could create a `rustup toolchain link` with a different name if this helps (see rust-qoriq-postinstall target in this file)

Reply:
@th0ma7, I've proposed a fix for this in #6477, and it seems to work with my test builds for `memcached`. Once I get approval, we can merge and rebase.

Reply:
> @th0ma7 I have reverted your change since `powerpc-unknown-linux-gnuspe` (and `1.82.0-powerpc-unknown-linux-gnuspe`) is a valid rust toolchain name.

This was just a really quick test I had pushed online as I thought the issue was dead simple to fix (which looks like not).